articles/005-fee-elimination.md

```diff
@@ -0,0 +1,125 @@
+ - $\frac{\text{Hold Duration} - \text{Time Elapsed}}{\text{Hold Duration}} = \frac{90 - 9}{90} = 0.9$
+ - Multiplied by the hold amount: $180 \times 0.9 = 162$
+
+The effective balance in that purse, at that point in time, is $1000 - 162 = 838 \ \text{CSPR}$
+
+Over the course of the hold's duration, this calculation gives us:
+| Hold Amount | Hold Period | Time Elapsed | Effective Hold |
+| --- | --- | --- | --- |
+| 180 | 90 | 1 | 178 |
+| 180 | 90 | 9 | 162 |
+| 180 | 90 | 45 | 90 |
+| 180 | 90 | 89 | 2 |
+
+### More about Gas holds 
+The duration of gas holds is defined [here](https://github.com/casper-network/casper-node/blob/feat-2.0/resources/production/chainspec.toml#L166) in the [casper-node](https://github.com/casper-network/casper-node) chainspec:
+
+```toml
+# If fee_handling is set to 'no_fee', the system places a balance hold on the payer
+# equal to the value the fee would have been. Such balance holds expire after a time
+# interval has elapsed. This setting controls how long that interval is. The available
+# balance of a purse equals its total balance minus the held amount(s) of non-expired
+# holds (see gas_hold_balance_handling setting for details of how that is calculated).
+#
+# For instance, if gas_hold_interval is 24 hours and 100 gas is used from a purse,
+# a hold for 100 is placed on that purse and is considered when calculating total balance
+# for 24 hours starting from the block_time when the hold was placed.
+gas_hold_interval = '24 hours'
+```
+
+
+### Preventing Exploitation
+The ultimate goal of any gas mechanism is to prevent exploitation of a network's resources. Aside from incentivizing validators, there is no fundamental reason to charge users for making transactions if their honesty can be guaranteed. By designing a system that disincentivizes wasteful transactions without charging a fee, resistance to exploitation can be maintained while allowing users to transact freely.
+
+However, any gas mechanism that doesn't charge users could be vulnerable to denial-of-service attacks. Provided a large enough bankroll, a user could deploy enough transactions to slow the network for the amount of time needed for his or her previous gas payments to unlock, and use these unlocked funds to deploy more transactions, and thus repeat the process ad infinitum. In this way, one could theoretically deploy infinite transactions, cycling through their locked and unlocked balances. 
+
+Attacking the network in this way is a challenge of economic feasibility, much like many other aspects of blockchains. To prevent an attack like this from taking place, it must be made prohibitively expensive to mount such an attack, with little to no incentive to the attacker. Casper's approach involves using a long locking period, combined with 16 second blocktimes. The Casper 2.0 mainnet is slated to roll out with a 30 day locking period, but if increased, the cost to attack would scale linearly.
```

Comment:
I would add a note here to emphasise that this 30-day hold period is just for example, and is not a certainty. The last conversation I had about this talked about a 90 day hold period. it is liable to change until close to the final rollout.